integration-web/src/main/resources/integration-web-log-messages.properties

```diff
@@ -1,32 +1,36 @@
-integration.web.integration.bridge.unavailable.solution = Check Integration Bridge health via health check. This can be done \
+integration.web.integration.bridge.unavailable=Integration Bridge temporarily unavailable due to connectivity issues.
+integration.web.integration.bridge.unavailable.solution=Check Integration Bridge health via health check. This can be done \
   by sending a GET request to [Integration Bridge URL]/integration/health
-integration.web.cannot.retrieve.whitelist = Cannot retrieve embedded {0} whitelist
-integration.web.cannot.retrieve.whitelist.solution = Check if your integration type on your WebHook URL is valid. \
+integration.web.cannot.retrieve.whitelist=Cannot retrieve embedded {0} whitelist
+integration.web.cannot.retrieve.whitelist.solution=Check if your integration type on your WebHook URL is valid. \
   by sending a GET request to [Integration Bridge URL]/integration/health
-integration.web.payload.couldnt.parse.payload = Couldn't parse the incoming payload for the instance {0} and configuration {1}
-integration.web.payload.couldnt.parse.payload.solution = Check if the integrity of the payload sent by your webhook and if it is \
+integration.web.payload.couldnt.parse.payload=Couldn't parse the incoming payload for the instance {0} and configuration {1}
+integration.web.payload.couldnt.parse.payload.solution=Check if the integrity of the payload sent by your webhook and if it is \
   on a valid format
-integration.web.webhook.configuration.unavailable = Configuration {0} unavailable!
-integration.web.webhook.configuration.unavailable.solution = Check if the configuration id on your WebHook URL is valid.
-integration.web.integration.unavailable = Integration {0} unavailable!
-integration.web.integration.unavailable.solution = Verify the health-check and make sure the integration have already \
+integration.web.webhook.configuration.unavailable=Configuration {0} unavailable!
+integration.web.webhook.configuration.unavailable.solution=Check if the configuration id on your WebHook URL is valid.
+integration.web.integration.unavailable=Integration {0} unavailable!
+integration.web.integration.unavailable.solution=Verify the health-check and make sure the integration have already \
    been deployed
 integration.web.integration.not.authorized=The informed integration (configuration id: {0}) does not support authorized API calls.
-integration.web.integration.not.authorized.solution=Provide an integration that implements the interface AuthorizedIntegration.
\ No newline at end of file
+integration.web.integration.not.authorized.solution=Provide an integration that implements the interface AuthorizedIntegration.
+integration.web.jwt.pod.url.malformed=The provided POD URL {0} is invalid/malformed. Send a correct one.
+integration.web.jwt.pod.url.unauthorized=The provided POD URL {0} is not the same used by the Inegration Brigde {1}. \
```

Comment:
Typo.

'The provided POD URL {0} is not the same used by the Integration Bridge {1}'